packages/netsuite-adapter/src/change_validator.ts

```diff
@@ -68,6 +68,11 @@ const nonSuiteAppValidators: ChangeValidator[] = [
   removeFileCabinetValidator,
 ]
 
+const changeErrorsToElementIDs = (changeErrors: readonly ChangeError[]): readonly string[] =>
```

Comment:
nit - a personal preference:
```suggestion
const changeErrorsToElementIDs = (changeErrors: ReadonlyArray<ChangeError>): ReadonlyArray<string> =>
```
anyway, it's not *that* important to use a readonly input&output IMO